src/components/reference-editor/index.js

```diff
@@ -0,0 +1,54 @@
+import React, { PropTypes } from 'react';
+import reactStamp from 'react-stamp';
+
+import EditorFactory from 'components/editor';
+
+import createMentionPlugin, { defaultSuggestionsFilter } from 'draft-js-mention-plugin';
+import createUndoPlugin from 'draft-js-undo-plugin';
+import createLinkifyPlugin from 'draft-js-linkify-plugin';
+
+import 'draft-js-mention-plugin/lib/plugin.css';
+import 'draft-js-undo-plugin/lib/plugin.css';
+import 'draft-js-linkify-plugin/lib/plugin.css';
+
+import MentionComponent from './mention-text';
+import SuggestionsFactory from './suggestions';
+
+export default function ({
```

Comment:
Two other options would be:

* have private methods on the component that will return the instances of plugins and `UpstreamEditor`, then either expect those or stub — but this seems a bit too involved
* export the actual instances of plugins and UpstreamEditor, and set expectations based on those

I've chosen to go the second road and pushed just now.